enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/TemporalAcceptanceTest.scala

```diff
@@ -475,6 +475,19 @@ class TemporalAcceptanceTest extends ExecutionEngineFunSuite with QueryStatistic
     }
   }
 
+  // Comparison of durations
+
+  test("should not allow comparing durations") {
```

Comment:
I thought this test was supposed to test the _return null_ runtime behaviour of comparing durations. Or is that another test?

Reply:
Why do you say "the same throw error behavior"? I don't understand. There was no other test for that. The default is that comparing durations gives and error and this test asserts on that.